### PR TITLE
add missing pod in the rbac

### PR DESCRIPTION
**Pull request by husnialhamdani**
**Description:** <Describe what has changed.>
Fixing a bug - add missing Pod resources in the RBAC

**Link to tracking Issue:** https://github.com/open-telemetry/opentelemetry-operator/issues/1679

### CODE REVIEW

**Review by jaronoff97 — APPROVED** [review]

**Review by frzifus — APPROVED** [review]

**Comment by husnialhamdani** [comment]
the `make bundle` command was not succesfully executed I guess, should I run that command and push the missing files?

**Comment by jaronoff97** [comment]
Yeah give that a shot. There are a few bad files in there for kustomize that you'll need to remove.

**Comment by husnialhamdani** [comment]
> Yeah give that a shot. There are a few bad files in there for kustomize that you'll need to remove.

How do I do/fix that?

File: bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -31,13 +31,13 @@ metadata:
...
     containerImage: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
-    createdAt: "2023-10-31T11:59:58Z"
+    createdAt: "2023-11-08T21:42:42Z"
     description: Provides the OpenTelemetry components, including the Collector
     operators.operatorframework.io/builder: operator-sdk-v1.29.0
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: github.com/open-telemetry/opentelemetry-operator
     support: OpenTelemetry Community
-  name: opentelemetry-operator.v0.88.0
+  name: opentelemetry-operator.v0.0.0
```
Comment (frzifus) [inline bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml:40]:
```suggestion
```

File: bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -31,13 +31,13 @@ metadata:
     categories: Logging & Tracing,Monitoring
     certified: "false"
     containerImage: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
-    createdAt: "2023-10-31T11:59:58Z"
+    createdAt: "2023-11-08T21:42:42Z"
```
Comment (frzifus) [inline bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml:34]:
```suggestion
```

File: bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -452,7 +453,7 @@ spec:
   minKubeVersion: 1.23.0
   provider:
     name: OpenTelemetry Community
-  version: 0.88.0
+  version: 0.0.0
```
Comment (frzifus) [inline bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml:456]:
```suggestion
```

File: config/manager/kustomization.yaml [hunk 4]
```diff
@@ -1,2 +1,7 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: ghcr.io/hua/opentelemetry-operator/opentelemetry-operator
```
Comment (frzifus) [inline config/manager/kustomization.yaml:7]:
```suggestion
```

**Review by frzifus — COMMENTED** [review]
To fix it manually, you can remove the marked lines. Sorry for that bad user experience :face_exhaling:

File: bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml [hunk 5]
```diff
@@ -344,7 +344,7 @@ spec:
                 - --zap-log-level=info
                 - --zap-time-encoding=rfc3339nano
                 - --feature-gates=+operator.autoinstrumentation.go,+operator.autoinstrumentation.nginx
-                image: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator:0.88.0
+                image: ghcr.io/hua/opentelemetry-operator/opentelemetry-operator
```
Comment (husnialhamdani) [inline bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml:347]: What about this?

File: bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml [hunk 6]
```diff
@@ -31,13 +31,12 @@ metadata:
...
     certified: "false"
     containerImage: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
-    createdAt: "2023-10-31T11:59:58Z"
+    createdAt: "2023-11-08T21:42:42Z"
     description: Provides the OpenTelemetry components, including the Collector
     operators.operatorframework.io/builder: operator-sdk-v1.29.0
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: github.com/open-telemetry/opentelemetry-operator
     support: OpenTelemetry Community
-  name: opentelemetry-operator.v0.88.0
```
Comment (jaronoff97) [inline bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml:40]: we want to keep this

File: bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml (hunk 5 above)
Comment (jaronoff97) [reply bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml:347]: we should revert this line

File: bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml (hunk 3 above)
Comment (jaronoff97) [reply bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml:455]: also revert this line

File: config/manager/kustomization.yaml (hunk 4 above)
Comment (jaronoff97) [reply config/manager/kustomization.yaml:7]: yep, also delete this

**Review by pavolloffay — CHANGES_REQUESTED** [review]
Please revert the changes referencing `hua` docker registry

**Comment by husnialhamdani** [comment]
Reverted back the bad files, could you please check again?

File: config/manager/kustomization.yaml [hunk 7]
```diff
@@ -1,2 +1,2 @@
 resources:
-- manager.yaml
+- manager.yaml
```
Comment (swiatekm-sumo) [inline config/manager/kustomization.yaml:2]: This looks like an unnecessary change. Otherwise LGTM.

Comment (husnialhamdani) [reply config/manager/kustomization.yaml]: Updated

**Review by swiatekm-sumo — APPROVED** [review]

**Review by pavolloffay — APPROVED** [review]

**Comment by husnialhamdani** [comment]
Thank you all for the review and approval
